tests/test_jiralib.py

```diff
@@ -0,0 +1,65 @@
+# This file is part of the github2jira project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2021 Red Hat, Inc.
+#
+
+import os
+
+from unittest import mock
+
+from unittest.mock import patch
+import pytest
+
+from github2jira.githublib import Issue
+import tests.common as common
+from tests.common import get_raw_issue
+
+
+def mockenv(**envvars):
+    return mock.patch.dict(os.environ, envvars)
+
+
+class MockedJIRA:
+    def __init__(self, options=None, basic_auth=None):
+        return
+
+
+@pytest.fixture
+def mocked_jira():
+    return MockedJIRA()
```

Comment:
I do not think you need this one.. it gives you nothing.

Reply:
we need to mock jira.JIRA so it won't try to connect the real jira in the constructor of jirablib.Jira